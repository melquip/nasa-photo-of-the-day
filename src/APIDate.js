import React from "react";
export function APIDate({ date }) {
	return (
		<p className="date">{date}</p>
	);
}
