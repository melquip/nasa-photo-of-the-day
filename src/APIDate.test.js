import React from "react";
import { APIDate } from "./APIDate";
import { create, act } from 'react-test-renderer';

it('renders without crashing', () => {
	let res;
	const today = new Date().toISOString().substr(0, 10);
	act(() => {
		res = create(<APIDate date={today} setDate={() => { }} today={today} />);
	})
	expect(res.toJSON()).toMatchSnapshot();
});

it('renders correctly', () => {
	let res;
	const today = new Date().toISOString().substr(0, 10);
	act(() => {
		res = create(<APIDate date={today} setDate={() => { }} today={today} />);
	})
	expect(res.toJSON()).toMatchSnapshot();
});
