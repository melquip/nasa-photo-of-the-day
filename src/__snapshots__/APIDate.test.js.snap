// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="date"
>
  <input
    max="2019-09-12"
    min="2018-01-01"
    onChange={[Function]}
    type="date"
    value="2019-09-12"
  />
</div>
`;

exports[`renders without crashing 1`] = `
<div
  className="date"
>
  <input
    max="2019-09-12"
    min="2018-01-01"
    onChange={[Function]}
    type="date"
    value="2019-09-12"
  />
</div>
`;
